pkg/agent/rollouts.go

```diff
@@ -142,14 +139,3 @@ func NewArgoRolloutsGetter() (argov1alpha1.RolloutsGetter, error) {
 
 	return argoClient, nil
 }
-
-func newK8sRestClient() (*rest.Config, error) {
```

Comment:
It looks like you moved this to `secrets.go`. Rather than do that should it move to a new file or probably the `utils.go` since it is shared in the package?

Reply:
Move to k8s.